core/trino-main/src/test/java/io/trino/operator/join/unspilled/TestHashJoinOperator.java

```diff
@@ -189,38 +191,40 @@ public void testInnerJoin(boolean parallelBuild, boolean probeHashEnabled, boole
         assertOperatorEquals(joinOperatorFactory, taskContext.addPipelineContext(0, true, true, false).addDriverContext(), probeInput, expected, true, getHashChannels(probePages, buildPages));
     }
 
-    @Test
-    public void testInnerJoinWithRunLengthEncodedProbe()
+    @Test(dataProvider = "typesProvider")
+    public void testInnerJoinWithRunLengthEncodedProbe(Type type, IntFunction<Object> valueProvider, IntFunction<Object> rleValueProvider)
     {
         TaskContext taskContext = createTaskContext();
 
         // build factory
-        RowPagesBuilder buildPages = rowPagesBuilder(false, Ints.asList(0), ImmutableList.of(VARCHAR))
```

Comment:
This seems quite complicated. Let's just switch to BIGINT and force default/bigint pages hash explicitly. VARCHAR was chosen here for convenience. 
We can leave a single test or two with multi channel columns (I think they already exist).

Reply:
Do you mean reverting changes in this class and just changing single-column tests to `BIGINT`?